loop.cpp

```diff
@@ -0,0 +1,55 @@
+#include "loop.h"
+
+#include <iostream>
+
+Loop::Loop(char **argv)
+{
+    this->players = new Player[NUMBER_OF_PLAYERS]();
+    for (unsigned short i = 0; i < NUMBER_OF_PLAYERS; i++)
+    {
+        this->players[i].Read(argv, i);
```

Comment:
A paraméterek feldolgozása nem a Player feladata. A jellemzőit a konstruktorban kellene átadni az inicializáláshoz.